src/boot.lisp

```diff
@@ -415,6 +415,16 @@
          (= (length obj) 5)   ;; 3
          (eq (car obj) 'std-instance)) )
 
+;;; js-object predicate
```

Comment:
Should we ensure we tag all CL objects with some special property and just test for that maybe?  This also works for now. Maybe use `not` instead of that `if`?